src/App.svelte

```diff
@@ -102,6 +131,10 @@ let answer = '';
               </option>
           {/each}
       </select>
+      <label for="fname">Short-Term Capital Gains:</label>
+      <input type="number" class = "income-entry" bind:value={capitalGainsShortTerm} on:input={calcTaxes}><br><br>
+      <label for="fname">Long-Term Capital Gains:</label>
+      <input type="number" class = "income-entry" bind:value={capitalGainsLongTerm} on:input={calcTaxes}><br><br>
```

Comment:
This is what I see when running the website locally:
![image](https://user-images.githubusercontent.com/55931391/224602966-5db0b2b8-323d-42f5-97d4-5e35d1eff657.png)

I feel this looks a bit scuffed. Can you move the two capital gains inputs right below the income input and put them side by side? Feel like that may look a bit better.